src/dmd/backend/tk.c

```diff
@@ -235,7 +175,7 @@ char *mem_fstrdup(const char *);
 
 #if MEM_NONE
 
-#define mem_inited      1
+//#define mem_inited      1
```

Comment:
Why a comment?

Reply:
`mem_inited` is defined in `mem.d` now, and declared `extern` in `tk.c`.  I didn't want any symbol conflicts.

Reply:
Why not remove it completely?

Reply:
:blush: Because sometimes I'm an idiot.

Reply:
😊